low-level/framework/utils/service_logging.py

```diff
@@ -19,130 +19,33 @@
  ****************************************************************************
 """
 
-import logging.handlers
-import time
+import sys
 import os
+import syslog
+from cortx.utils.log import Log
+
+
+def init_logging(service_name, file_path, log_level="INFO"):
```

Comment:
> **cdeshmukh**: legacy DCS, there used to be another module along with SSPL daemon. we can safely rename that in new code, just keep service_name

Done

Reply:
@jjxsg , While trying out with this change, logs are suppressed in sspl.log also same is writing into /var/log/messages.